spec/features/organisation_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+RSpec.feature 'Organisation', type: :feature do
+  scenario 'Organisation creation' do
+  user = FactoryBot.create(:user, name: 'Jane Doe', date_of_birth: Date.new,  line1: 'line 1', phone_number: '123', postcode: 'W11AA', townCity: 'London', county: 'London')
+  login_as(user, :scope => :user)
+  visit '/'
+  expect(page).to have_text 'Start a new project'
+  click_link_or_button 'Start a new project'
+  click_link_or_button 'Start Now'
+  choose 'Registered charity'
+  click_link_or_button 'Save and continue'
+  fill_in 'organisation[charity_number]', with: '123'
+  click_link_or_button 'Save and continue'
+  set_address(title_field = 'Organisation name')
+  check 'Female led'
+  click_link_or_button 'Save and continue'
+  fill_in 'Full name', match: :first, with: 'Jane Doe'
+  fill_in 'Email address', match: :first, with: 'test@example.com'
+  fill_in 'Phone number', match: :first, with: '123'
+  click_link_or_button 'Save and continue'
+  expect(page).to have_text('Check your answers')
+  expect(page).to have_text('Registered charity')
+  expect(page).to have_text('Female led')
```

Comment:
We should check that the charity number, address and legal signatory name exist on this page too.

```suggestion
  expect(page).to have_text('Female led')
  expect(page).to have_text('123')
  expect(page).to have_text('4 Barons Court Road')
  expect(page).to have_text('Jane Doe')
```